tests/core/conftest.py

```diff
@@ -335,7 +335,7 @@ def wait_for_condition(condition_type, status, client, obj, timeout=45):
         delta = time.time() - start
         if delta > timeout:
             msg = 'Timeout waiting for [{}:{}] for condition after {}' \
-                ' seconds'.format(obj.type, obj.id, delta)
+                ' seconds\n {}'.format(obj.type, obj.id, delta, obj)
```

Comment:
Should just throw the condition_type and status in there as well. This makes me think we should create our own exception to add info to

Reply:
Since we dump the entire object those will also be in there. Do you have them before the newline?